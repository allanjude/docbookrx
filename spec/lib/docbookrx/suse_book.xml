<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
                 type="text/xml" 
                 title="Profiling step"?>
<!DOCTYPE book
[
  <!ENTITY % entities SYSTEM "entity-decl.ent">
    %entities;
]>
<book version="5.0" xml:lang="en" xml:id="book.suma.getting-started"
    xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
    xmlns:xlink="http://www.w3.org/1999/xlink">
    <?dbjsp dir="getting-started/en-US" ?>
    <?dbjsp filename="index.jsp"?>

    <info>
        <title>Getting Started</title>

        <author>
            <personname>
                <firstname>Klaus</firstname>
                <surname>KÃ¤mpf</surname>
            </personname>
        </author>
        <editor>
            <personname>
                <firstname>&susemgr; Team</firstname>
            </personname>
        </editor>

        <productname>&productname;</productname>
        <productnumber>&productnumber;</productnumber>

        <date>
            <?dbtimestamp format="Y-m-d"?>
        </date>

    </info>
</book>
