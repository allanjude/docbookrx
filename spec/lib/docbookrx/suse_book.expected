= Getting Started

Klaus KÃ¤mpf

            
        
:doctype: book
:sectnums:
:toc: left
:icons: font
:experimental: